admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -1622,7 +1622,7 @@ var priceCalculation = (function() {
     },
     'impactTaxInclude': function(obj) {
       var price = parseFloat(obj.val().replace(/,/g, '.'));
-      var targetInput = obj.parent().parent().parent().find('input.attribute_priceTI');
+      var targetInput = obj.parent().parent().parent().parent().find('input.attribute_priceTI');
```

Comment:
I talked to @antho-girard about this on Gitter, he'll try to find a more durable fix :)